app/views/comments/_comment.html.erb

```diff
@@ -24,6 +24,7 @@
     data-path="<%= commentable&.path %>/comments/<%= comment.id_code_generated %>"
     data-comment-author-id="<%= comment_user_id_unless_deleted comment %>"
     data-content-user-id="<%= comment_user_id_unless_deleted comment %>">
+    <a name="comment-<%= comment.id_code %>" style="position: absolute; top: -8px;">&nbsp;</a>
```

Comment:
This is an emtpy/transparent anchor tag that will help link to an article's comment. It's positioned so that the sticky navbar doesn't touch the comment's card border